modules/universalId.js

```diff
@@ -0,0 +1,406 @@
+/**
+ * This module adds Universal ID support to prebid.js
+ */
+import { ajax } from 'src/ajax';
+import { config } from 'src/config';
+import events from 'src/events';
+import * as utils from 'src/utils';
+import find from 'core-js/library/fn/array/find';
+import { gdprDataHandler } from 'src/adaptermanager';
```

Comment:
Just FYI, as part of #3435, all import paths inside `modules` or `src` directories need to be relative now since applications using Prebid.js through npm won't be able to correctly resolve where `src` is.

So this line, for instance, should be `import { gdprDataHandler } from '../src/adaptermanager';`.